src/global_sensitivity/sobol_sensitivity.jl

```diff
@@ -44,58 +45,90 @@ function gsa(f, method::Sobol, A::AbstractMatrix, B::AbstractMatrix;
         all_y = multioutput ? reduce(hcat, _y) : _y
     end
 
+    Eys = []
```

Comment:
instability

Reply:
It gets weird because of the multioutput and non-multioutput branches, I'll try to come up with something that works, I don't think it's that big an issue though.